ssh1

```diff
@@ -95,14 +95,15 @@ case $aws in
     ;;
 esac
 
+AWS_CMD="aws --region $aws"
 # Create the test table "usertable" (deleting it first if it exists)
 # FIXME: need to delete the table without snapshots, which just wastes
 # time on the server! If we can't configure Scylla properly, at least
 # ssh to the server and delete the snapshots :-(
-aws dynamodb delete-table \
+$AWS_CMD --region $aws dynamodb delete-table \
```

Comment:
No need to include --region $aws both in each command and inside AWS_CMD itself.

Reply:
That was left here by mistake